src/Components/Auth/Usercard/Usercard.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import firebase from "firebase/app";
+import "firebase/auth";
+import { FirebaseAuthProvider, FirebaseAuthConsumer, IfFirebaseAuthedAnd, IfFirebaseUnAuthed } from "@react-firebase/auth";
+import { Media } from 'reactstrap';
+import GoogleLogin from '../Login/Login.js';
+import Signout from '../Signout/Signout.js';
+
+const firebaseConfig = {
```

Comment:
We should make a new file and import the class from there rather than duplicating the constant so many times